src/styles/Exercise.css

```diff
@@ -0,0 +1,3 @@
+span {
+    font-weight: bold;
+}
```

Comment:
This applies to ALL spans. This rule needs to be more specific, perhaps using a class.